app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-form-tracker.js

```diff
@@ -89,9 +89,34 @@ window.GOVUK.Modules = window.GOVUK.Modules || {};
       input.section = labelText
 
       var isTextField = inputTypes.indexOf(inputType) !== -1 || inputNodename === 'TEXTAREA'
+      var conditionalField = elem.closest('.govuk-checkboxes__conditional')
```

Comment:
Just checking, are you aware of the `data-ga4-form-include-input` option on the form tracker, that can be applied to inputs (like the conditional ones) that should be ignored?

Reply:
I thought that this marked an element to be exempt from redaction rather than it being ignored?

Reply:
Ah yes you're right.

Reply:
I'm pretty keen to rewrite this tracker using more contemporary JS, as myself and other frontend developers that I talked through this PR with on a call found the existing code difficult to parse (such as it using `for` and not `forEach`). Since this PR is just to add new functionality, I thought that changing the style was out of scope.